.gitignore

```diff
@@ -0,0 +1 @@
+.idea
```

Comment:
In my opinion, it is better to handle editor-specific files in a global `.gitignore`. 

Basically, every project we open with IntelliJ will have `.idea` or `.iml` files, but there is no need to handle them on the project level as these are developer-specific files, not project-specific (say, some people use VS Code or Eclipse).

I think the better way would be to create a `~/.gitignore_global` file and import it to git using `git config --global core.excludesfile ~/.gitignore_global`. This way Git will ignore `.idea` folders in all projects, no matter if they have it specified explicitly or not. You can read more about this [here](https://help.github.com/en/github/using-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer).

What do you think about this approach?

Reply:
Good point. I have never heard about this